modules/peers.js

```diff
@@ -286,6 +287,31 @@ Peers.prototype.sandboxApi = function (call, args, cb) {
 	sandboxHelper.callMethod(Peers.prototype.shared, call, args, cb);
 };
 
+/**
+ * Calculates consensus for as a ratio active to matched peers.
+ * @param {Array<Peer>} matched - peers with same as system broadhash
+ * @param {Array<Peer>} active - active peers (with connected state)
+ * @returns {number|undefined} - return consensus or undefined if config.forging.force = true
+ */
+Peers.prototype.getConsensus = function (matched, active) {
+
+	if (library.config.forging.force) {
+		return undefined;
+	}
+
+	active = active || __private.getByFilter({state: Peer.STATE.CONNECTED, normalized: false});
+	matched = matched || active.filter(function (peer) {
+		return peer.broadhash === modules.system.getBroadhash();
+	});
+
+	active = active.slice(0, constants.maxPeers);
+	matched = matched.slice(0, constants.maxPeers < active.length ? constants.maxPeers : active.length);
```

Comment:
`constants.maxPeers < active.length` - it will never evaluate to `true`, so:
`active = matched.length < active.length ? matched.length : active.length;`

Reply:
or `matched = Math.min(matched.length, active.length);`